### PR TITLE
Add convert const to let

### LINKED ISSUE #22473: Codefix: convert const to let
**TypeScript Version:**  2.8.0-dev.20180308

**Search Terms:**
const let codefix

**Code**
```ts
const x = 42;

// lots of code

x/*1*/ = 75;
```

**Expected behavior:**
Error with codefix offered at 1 to convert the declaration from `const` to `let`

**Actual behavior:**
Just an error, no codefix

**Related Issues:**
The exact opposite of #13133

### PR DESCRIPTION
**Pull request by fuafa**
Fixes #22473

### CODE REVIEW

**Comment by orta** [comment]
Hi @fuafa thanks for the PR - perfect. After reading the original issue there was a mention about:

> Also you will need to add handling for the readonly property case, where the readonly modifier would be removed.

But we don't need to handle that because that's now a separate diagnostic error (and so this fixit won't be included)

File: src/services/codefixes/convertConstToLet.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/* @internal */
+namespace ts.codefix {
+    const fixId = "fixConvertConstToLet";
+    const errorCodes = [Diagnostics.Cannot_assign_to_0_because_it_is_a_constant.code];
+
+    registerCodeFix({
+        errorCodes,
+        getCodeActions: context => {
+            const { sourceFile, span, program } = context;
+            const variableStatement = getVaribleStatement(sourceFile, span.start, program);
```
Comment (orta) [inline src/services/codefixes/convertConstToLet.ts]:
```suggestion
            const variableStatement = getVariableStatement(sourceFile, span.start, program);
```

File: src/services/codefixes/convertConstToLet.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        getCodeActions: context => {
+            const { sourceFile, span, program } = context;
+            const variableStatement = getVaribleStatement(sourceFile, span.start, program);
+            const changes = textChanges.ChangeTracker.with(context, t => doChange(t, sourceFile, variableStatement));
+            return [createCodeFixAction(fixId, changes, Diagnostics.Convert_const_to_let, fixId, Diagnostics.Convert_const_to_let)];
+        },
+        fixIds: [fixId]
+    });
+
+    function getVaribleStatement(sourceFile: SourceFile, pos: number, program: Program) {
```
Comment (orta) [inline src/services/codefixes/convertConstToLet.ts]:
```suggestion
    function getVariableStatement(sourceFile: SourceFile, pos: number, program: Program) {
```

**Comment by fuafa** [comment]
Thanks @orta, sorry for the late response, typo fixed.

**Comment by orta** [comment]
Looks good, thanks 👍

**Pull request merged**